epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModelTouchCallback.java

```diff
@@ -206,26 +206,30 @@ protected void onChildDraw(Canvas c, RecyclerView recyclerView, EpoxyViewHolder
       float dX, float dY, int actionState, boolean isCurrentlyActive) {
     super.onChildDraw(c, recyclerView, viewHolder, dX, dY, actionState, isCurrentlyActive);
 
-    EpoxyModel<?> model = viewHolder.getModel();
```

Comment:
can you limit the try /catch to just `viewHolder.getModel()` to reduce scope?

Reply:
done